usr/bin/nxdk-pkg-config

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "$0 $@" >> /tmp/$(basename $0).log
```

Comment:
This logging should be conditional and improved, but it should probably be part of these wrappers to ease debugging of complicated build-systems